### PR TITLE
[LEAF-1284] Refresh directory UI and process update 

### PR DESCRIPTION
**Pull request by thomasnguyen95**
Runs php script in background
Updates UI with no loading spinner and basic static text
Fixes parametrized queries for refreshing employee

### CODE REVIEW

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5034127&bid=16583761)

File: LEAF_Nexus/sources/System.php [hunk 1]
```diff
@@ -199,4 +199,17 @@ public function removeReportTemplate($in)
...
+    public function refreshOrgchartEmployees()
+    {
+        $memberships = $this->login->getMembership();
+        if (!isset($memberships['groupID'][1]))
+        {
+            return 'Admin access required';
+        }
+        
+        header('Cache-Control: no-cache');
+        exec('php ../scripts/refreshOrgchartEmployees.php &', $output);
```
Comment (patrickcgs) [inline LEAF_Nexus/sources/System.php]: @dastan-rahimi 

Will the use of the relative path here work in production? Note that existing pattern from mailer uses variable for path directory that is based on php variable __FILE__:

https://github.com/VHAINNOVATIONS/LEAF/blob/master/LEAF_Request_Portal/Email.php#L177

cc: @thomasnguyen95

Comment (thomasnguyen95) [reply LEAF_Nexus/sources/System.php]: Yes it will work

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5034127&bid=16583761)

**Pull request merged**